flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/src/test/java/org/apache/flink/cdc/pipeline/tests/RouteE2eITCase.java

```diff
@@ -719,6 +718,11 @@ public void testMergeTableRouteWithTransform() throws Exception {
 
     @Test
     public void testReplacementSymbol() throws Exception {
+        String defaultFlinkProperties = getFlinkProperties(flinkVersion);
+        overrideFlinkProperties(
+                defaultFlinkProperties.replace(
+                        "execution.checkpointing.interval: 300",
+                        "execution.checkpointing.interval: 10000"));
```

Comment:
I've tried to run this locally and `overrideFlinkProperties` seems not working since this environment variable will only take effect once when the container started in `before()` hook function.

<img width="1145" alt="截圖 2024-08-28 08 50 02" src="https://github.com/user-attachments/assets/c2402109-451f-49be-859f-493bf2e31a49">

Reply:
good catch， looks like we use overrideFlinkProperties wrongly